src/settings.rs

```diff
@@ -29,9 +27,13 @@ impl SettingsFile {
         }
     }
 
-    fn write_settings(&self) -> Result<()> {
-        let s = self.cache.borrow().as_ref().unwrap().clone();
-        utils::write_file("settings", &self.path, &s.stringify())?;
+    fn write_settings(&self) -> anyhow::Result<()> {
```

Comment:
Hmmm looks like we're already using `anyhow::Result` as `Result` earlier in the file? Same for 2 other places...